pkg/nsx/services/inventory/types.go

```diff
@@ -48,6 +48,12 @@ const (
 	InventoryStatusUp      = "UP"
 	InventoryStatusDown    = "DOWN"
 	InventoryStatusUnknown = "UNKNOWN"
+
+	NcpLbError     = "ncp/error.loadbalancer"
+	NcpLbPortError = "ncp/error.loadbalancer.unrealized_ports"
+	NcpLbEpError   = "ncp/error.loadbalancer_endpoints"
+	NcpDlbError    = "ncp/error.distributed_loadbalancer"
+	NcpSnatError   = "ncp/error.snat"
```

Comment:
I see there is another error `ncp/error.vs_access_log` might be added to Service, would you know if it is requried?

Reply:
OK, added.